device-types/Cisco/MDS-C9148S.yaml

```diff
@@ -0,0 +1,138 @@
+---
+manufacturer: Cisco
+model: MDS 9148S
```

Comment:
If you run a `show version` or other applicable command, what is the model that it says it is? Is this all that it says?